packages/shader-shaderlab/src/shaders/shadingPBR/BSDF.glsl

```diff
@@ -418,21 +425,20 @@ vec3 envBRDFApprox(vec3 specularColor, float roughness, float dotNV ) {
 
 void initBSDFData(SurfaceData surfaceData, out BSDFData bsdfData){
     vec3 albedoColor = surfaceData.albedoColor;
-    vec3 specularColor = surfaceData.specularColor;
     float metallic = surfaceData.metallic;
     float roughness = surfaceData.roughness;
-    float f0 = surfaceData.f0;
 
-    #ifdef IS_METALLIC_WORKFLOW
-        bsdfData.diffuseColor = albedoColor * ( 1.0 - metallic );
-        bsdfData.specularColor = mix( vec3(f0), albedoColor, metallic );
-    #else
-        float specularStrength = max( max( specularColor.r, specularColor.g ), specularColor.b );
-        bsdfData.diffuseColor = albedoColor * ( 1.0 - specularStrength );
-        bsdfData.specularColor = specularColor;
-    #endif
+    vec3 dielectricF0 = vec3(pow2( (surfaceData.IOR - 1.0) / (surfaceData.IOR  + 1.0) ));
+    dielectricF0 = min(dielectricF0 * surfaceData.specularColor , vec3(1.0)) * surfaceData.specularIntensity;
+    float dielectricF90 = surfaceData.specularIntensity;  
```

Comment:
The properties of this plugin are designed for dielectrics. Is there any ambiguity in the API?